assets/js/system-status.ts

```diff
@@ -7,65 +7,54 @@ window.addEventListener('DOMContentLoaded', () => {
   );
 
   const $operationalIcons = document.querySelectorAll('.operational');
-  const $degradedPerformanceIcons = document.querySelectorAll(
-    '.degraded-performance'
-  );
-  const $partialServiceDisruptionIcons = document.querySelectorAll(
-    '.partial-service-disruption'
-  );
-  const $plannedMaintenanceIcons = document.querySelectorAll(
-    '.planned-maintenance'
-  );
-  const $serviceDisruptionIcons = document.querySelectorAll(
-    '.service-disruption'
-  );
-  const $securityEventIcons = document.querySelectorAll('.security-event');
+  const $warningIcons = document.querySelectorAll('.degraded-performance');
   const $allSystemStatusIcons = document.querySelectorAll('.status-icon');
 
   type SystemStatus = {
-    icons: NodeList;
+    icon: NodeList;
     message: string;
+    title: string;
   };
 
   // https://kb.status.io/developers/status-codes/
-  const status: Record<string, SystemStatus> = {
-    DEGRADED_PERFORMANCE: {
-      icons: $degradedPerformanceIcons,
-      message:
-        'Degraded Performance - We are currently experiencing ' +
-        'degraded performance in some of our web services.',
-    },
-    OPERATIONAL: {
-      icons: $operationalIcons,
+  const status: Record<number, SystemStatus> = {
+    100: {
+      icon: $operationalIcons,
       message: '',
+      title: 'Operational',
     },
-    PARTIAL_SERVICE_DISRUPTION: {
-      icons: $partialServiceDisruptionIcons,
-      message:
-        'Partial Service Disruption - Some of our web services ' +
-        'are temporarily unavailable.',
+    200: {
+      icon: $warningIcons,
+      message: 'We are currently undergoing some scheduled maintenance.',
+      title: 'Planned Maintenance',
     },
-    PLANNED_MAINTENANCE: {
-      icons: $plannedMaintenanceIcons,
+    300: {
+      icon: $warningIcons,
+      // eslint-disable-next-line max-len
```

Comment:
I think `// eslint-disable-next-line max-len` should be moved above `'We are currently experiencing degraded performance in some of our web services.',`.

Alternatively, you could do

```
'We are currently experiencing degraded performance in some of our' + 
'web services.',
```